.github/workflows/unit.yml

```diff
@@ -79,7 +79,7 @@ jobs:
         sudo rm /etc/apt/sources.list.d/* && sudo apt-get update
         sudo apt-get --no-install-recommends install -y ninja-build lcov
     - name: Build Simulation
-      run: ./script/cmake-build simulation
+      run: ./script/cmake-build simulation -DOT_MULTIPAN_RCP=ON
```

Comment:
@hemanth-silabs This is resolved.  Note that Nick made a change to separate it out into two different jobs since I made this comment. @bertoldi-silabs Please resolve this conversation.

Note: There is still an outstanding question of how these artifacts will look for both jobs / how they will affect code coverage, but I think we can get advice from the OpenThread community on that.